### ISSUE TITLE
Is the input for detector in the form of BGR

### ISSUE THREAD

**Issue by Xinxinatg** [yl305237731/flexible-yolov5#76]
I am trying to use the model in this repo for a kaggle competition, whereas the format of the image is confusing. I tested the model with cv2.imread and it ran well, but it didn't work with the image provided by the kaggle competition. this is the discussion page I posted on the kaggle website with the tunable notebook

**Comment by Xinxinatg** [comment]
This is the kaggle notebook https://www.kaggle.com/timxinxinpeng/flexible-yolo-asking-for-help

**Comment by yl305237731** [comment]
line 598 in datasets.py 
 ''img = cv2.imread(path)  # BGR''

**Comment by Xinxinatg** [comment]
> 

Thansk for replying! I converted the format to RGB in the Kaggle submission, but still only got 0.000 at the learderboard. Actually the model from your repo has achieved the best performance during training among all the models I trained, the issue is that it cannot generate result at kaggle submission. Surprisingly there has been no one using flexible-yolo for kaggle competion, otherwise I can have a look at other people's example. It's a shame that such good model from your repo couldn't contribute to the competion.

**Comment by yl305237731** [comment]
@Xinxinatg 
@Xinxinatg 
for train: 
"""
 img = cv2.imread(path) # BGR
 .......do something...
 img = img[:, :, ::-1].transpose(2, 0, 1) # RGB
 img = np.ascontiguousarray(img)
"""

for test,  see detect_image in scripts/detector.py.

"""
        im0s = image  # BGR 
        img = letterbox(im0s, new_shape=self.img_size)[0]  # BGR
        img = img[:, :, ::-1].transpose(2, 0, 1)  # RGB
        img = np.ascontiguousarray(img)
        img = torch.from_numpy(img).to(self.device)
        img = img.float()
        img /= 255.0
        if img.ndimension() == 3:
            img = img.unsqueeze(0)
        pred = self.model(img)[0]

"""

the input image format for network is RGB.

**Comment by Xinxinatg** [comment]
Thanks a lot for your help! I just figured there might be clash between the environment for running flexible yolov5 and Kaggle competition. But the model cofig I trained with flexible yolov5 has achieved extraordinary good results. Are there any chance you know how to convert the following flexible yolov5 config to official yolov 5 one? 

backbone:
  type: YOLOv5
  version: s # x, m, l ,x
  focus: True
  with_C3TR: True
head:
  nc: 1
  stride: [8.0, 16.0, 32.0]
  anchors:
    - [10,13, 16,30, 33,23]  # P3/8
    - [30,61, 62,45, 59,119]  # P4/16
    - [116,90, 156,198, 373,326]  # P5/32
